lib/tls.js

```diff
@@ -53,7 +53,8 @@ exports.convertNPNProtocols = function(protocols, out) {
   }
   // If it's already a Buffer - store it
   if (protocols instanceof Buffer) {
-    out.NPNProtocols = protocols;
+    // copy new buffer not to be modified by user
```

Comment:
Style nit: can you at least capitalize the comment and preferably punctuate it too?  (I know the other comments don't add punctuation but why settle for average?)